alicloud/resource_alicloud_ddoscoo_instance.go

```diff
@@ -74,11 +74,10 @@ func resourceAliCloudDdoscooInstance() *schema.Resource {
 				ValidateFunc: StringInSlice([]string{"coop"}, false),
 			},
 			"product_plan": {
-				Type:         schema.TypeString,
-				Optional:     true,
-				ForceNew:     true,
-				Computed:     true,
-				ValidateFunc: StringInSlice([]string{"0", "1", "2", "3"}, false),
+				Type:     schema.TypeString,
```

Comment:
=== RUN   TestAccAliCloudDdoscooInstance_basic0
    provider_test.go:149: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudDdoscooInstance_basic0 (115.50s)
PASS

=== RUN   TestAccAliCloudDdoscooInstance_basic0_twin
    provider_test.go:149: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudDdoscooInstance_basic0_twin (12.82s)
PASS

=== RUN   TestAccAliCloudDdoscooInstance_basic0_intl
    provider_test.go:149: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudDdoscooInstance_basic0_intl (145.69s)
PASS

=== RUN   TestAccAliCloudDdoscooInstance_basic1_dip
    provider_test.go:143: Skipping unsupported region eu-central-1. Supported regions: [ap-southeast-1]. Using ap-southeast-1 as this test region
--- PASS: TestAccAliCloudDdoscooInstance_basic1_dip (27.64s)
PASS

=== RUN   TestAccAliCloudDdoscooInstance_basic1_dip_intl
    provider_test.go:143: Skipping unsupported region eu-central-1. Supported regions: [ap-southeast-1]. Using ap-southeast-1 as this test region
--- PASS: TestAccAliCloudDdoscooInstance_basic1_dip_intl (25.58s)
PASS